index.bs

```diff
@@ -1219,16 +1226,14 @@ credential.
 
 <pre class="idl">
     dictionary PublicKeyCredentialUserEntity : PublicKeyCredentialEntity {
-        BufferSource   id;
+        ArrayBuffer    id;
```

Comment:
`BufferSource` is preferred here, according to the note in https://heycam.github.io/webidl/#idl-buffer-source-types:

> When designing APIs that take a buffer, it is recommended to use the BufferSource typedef rather than ArrayBuffer or any of the view types.

Reply:
Fixed!